extensions/fun.py

```diff
@@ -80,6 +80,10 @@ async def matheistmagic(self, ctx):
     async def error(self, ctx):
         await ctx.send('{0.mention} Claßen: Error heißt ich weigere mich!'.format(ctx.author))
 
+    @commands.command(hidden=True)
+    async def spam(self, ctx):
+        await ctx.send('{0.mention} Selber spam!'.format(ctx.author))
+
```

Comment:
na, do we really need that? I thought you implemented that one cannot spam messages

Reply:
This is a fun command